test-utils/generators/ViewModelGenerators.scala

```diff
@@ -207,23 +207,23 @@ trait ViewModelGenerators extends BasicGenerators {
 
   def furtherDetailsViewModel: Gen[FurtherDetailsViewModel] =
     for {
-      title <- nonEmptyMessage
+      title <- nonEmptyMessage(30)
       contents <- nonEmptyDisplayMessage
     } yield {
       FurtherDetailsViewModel(title, contents)
     }
 
   def radioListRowViewModelGen: Gen[RadioListRowViewModel] =
     for {
-      content <- nonEmptyMessage
-      value <- nonEmptyString
+      content <- nonEmptyMessage(30)
```

Comment:
trimmed down the generated string values to make them more digestable when debugging